examples/statistics/hist.py

```diff
@@ -46,7 +46,7 @@
 # edit the histogram to our liking. Let's change the color of each bar
 # based on its y value.
 
-fig, axs = plt.subplots(1, 2, figsize=(10, 5), tight_layout=True)
+fig, axs = plt.subplots(1, 2, tight_layout=True)
```

Comment:
I think it was intentional just from an aesthetic standpoint, but I'm gonna chock this up to my OCD and say I'm +1 on changing it back since it looks dumb hanging off the side of the page like that :-)